obspy/signal/tests/test_filter.py

```diff
@@ -256,12 +256,12 @@ def test_bandpass_high_corner_at_nyquist(self):
                                     df=df, corners=corners)
                     got3 = bandpass(data, low_corner, nyquist + 1.78,
                                     df=df, corners=corners)
-                    assert len(w) == 3
                     for w_ in w:
                         assert 'Selected high corner frequency ' in \
                             str(w[0].message)
                         assert 'Applying a high-pass instead.' in \
                             str(w[0].message)
+                    assert len(w) == 3
```

Comment:
This does not make sense to me; firstly, moving the assert should not change anything? Secondly, the asserts in the loop use `w[0].message`, meaning it checks the message from _the same warning_ every time through the loop. I'm pretty sure it should be checking `w_.message`.

Reply:
I moved the assert down to see the message of the 3 unexpected warnings. I could not reproduce this locally. `w_.message` - true, thanks ... corrected in the last commit.